setup.py

```diff
@@ -794,6 +800,16 @@ def run(self):
     elif sys.platform == 'darwin':
         ext_modules.append(Extension('borg.platform.darwin', [platform_darwin_source]))
 
+    try:
+        ncpus = __import__("multiprocessing").cpu_count()
```

Comment:
same here, why `__import__`?

Reply:
```
>>> import multiprocessing as mp
>>> mp.cpu_count()
4
```
This is on my laptop. It has 1 CPU (socket), 2 cores. Due to HT, it can do 4 threads in parallel.

So, even if the call is named cpu_count, we can be more precise here and call it cpu_threads maybe?

Reply:
Good point. To match the intent maybe cythonize_threads or n_cythonize_threads would be good names? logical_cpu_count might be a good name to explain the value.

Reply:
i'ld still go for cpu_threads and also move it to the top of the file, so it is available for whatever it needs.

Reply:
done